tests/failing-symbolic.pyk

```diff
@@ -20,7 +20,6 @@ tests/specs/functional/lemmas-no-smt-spec.k
 tests/specs/functional/lemmas-spec.k
 tests/specs/functional/merkle-spec.k
 tests/specs/functional/storageRoot-spec.k
-tests/specs/mcd/functional-spec.k
```

Comment:
Why this test in particular?

Reply:
I just happened to use it to test the non-foundry test parallelization and found that it was passing with the pyk prover.